package.json

```diff
@@ -46,6 +52,7 @@
     "typeorm": "^0.2.28"
   },
   "devDependencies": {
+    "@microsoft/microsoft-graph-types": "^1.23.0",
```

Comment:
why only a dev dependency?

Reply:
types are development dependency, they are not needed for production.